### DISCUSSION TITLE
How to send email using  front end sdk like pb.send('email',body) something like that...

### DISCUSSION THREAD

**Discussion by ajayyy123** [pocketbase/pocketbase#2808] [Q&A]
Is there any way to send emails to to anyone using front end or pocket base sdk . I am already aws smtp service .

**Comment by ganigeorgiev** [comment]
There is no mail web API at the moment.

Eventually it may get implemented in the future as part of https://github.com/pocketbase/pocketbase/issues/2111, but even then it will require admin authentication.
